helm/prometheus-meta-operator/templates/prometheus-rules/systemd.workload-cluster.rules.yml

```diff
@@ -10,16 +10,6 @@ spec:
   groups:
   - name: systemd
     rules:
-    - alert: WorkloadClusterSystemdUnitFailed
-      annotations:
-        description: '{{`Systemd unit {{ $labels.name }} is failed on {{ $labels.instance }}.`}}'
-      expr: node_systemd_unit_state{cluster_type="workload_cluster",name!~"coreos-metadata-sshkeys@core.service|debug-tools.service|k8s-addons.service|locksmithd.service|sshd.+service|systemd-coredump.+service|systemd-networkd.service|update-engine.service|update-engine-stub.service", state="failed"} == 1
```

Comment:
Any idea why this is so expensive? Considering we still have the ctitical ones below

Reply:
My bet is on the size of the regex. In one WC prometheus, `WorkloadClusterSystemdUnitFailed` takes 9.8s, `WorkloadClusterCriticalSystemdUnitFailed` takes 299ms